src/ec/packages/ec-component-blockquote/ecl-blockquote.html.twig

```diff
@@ -40,7 +40,9 @@
 
 <blockquote class="{{ _css_class }}"{{ _extra_attributes|raw }}>
   <p class="ecl-blockquote__body">{{ _citation }}</p>
+  {% if _author  %}
```

Comment:
As mentioned before, being this a mandatory element there is not reason for this check, we have to enforce some check on these mandatory elements but not this way.